src/components/ebay-progress-bar/index.marko

```diff
@@ -0,0 +1,7 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+static var ignoredAttributes = ["class"];
+
+<progress
```

Comment:
Are any `aria-labels` required for this to announce the progress?
If so can you add `a11yText` input.

Reply:
I don't think so, because we're using HTML progress.
Looking [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role) they just have `aria-labels` for things that the progress bar already has.